static/sass/styles.scss

```
@@ -700,3 +701,7 @@ $p-small-lh-diff: map-get($line-heights, default-text) - map-get($line-heights,
 #latest-articles article:first-child hr {
   display: none;
 }
+
+.p-navigation__tagged-logo .p-navigation__logo-tag {
+  background-color: #E95420;
```

Would it be good to reference the variable here instead of the hard-coded value?
```suggestion
  background-color: $color-accent;
```